pyteal/ast/abi/array_static.py

```diff
@@ -127,7 +152,7 @@ def length(self) -> Expr:
         Returns:
             A PyTeal expression that represents the static array length.
         """
-        return Int(self.type_spec().length_static())
+        return Int(int(self.type_spec().length_static()))
```

Comment:
nit: do we need this `int` specify? or it is for python failing to infer that it is `StaticArrayTypeSpec`?